.github/workflows/ci.yml

```diff
@@ -46,12 +46,6 @@ jobs:
     steps:
     - name: Checkout code
       uses: actions/checkout@v2
-    - name: Set up the environment
-      run: |
-        cp .env.example .env
-        pip3 install podman-compose
-    - name: Build using podman-compose
-      run: podman-compose build
```

Comment:
Had to drop this because there's a [bug](https://github.com/containers/podman-compose/issues/586) in podman-compose that renders it useless anyway.